src/components/structures/login/ForgotPassword.js

```diff
@@ -78,30 +79,27 @@ module.exports = React.createClass({
         ev.preventDefault();
 
         if (!this.state.email) {
-            this.showErrorDialog("The email address linked to your account must be entered.");
+            this.showErrorDialog(_t('The email address linked to your account must be entered.'));
         }
         else if (!this.state.password || !this.state.password2) {
-            this.showErrorDialog("A new password must be entered.");
+            this.showErrorDialog(_t('A new password must be entered') + ".");
         }
         else if (this.state.password !== this.state.password2) {
-            this.showErrorDialog("New passwords must match each other.");
+            this.showErrorDialog(_t('New passwords must match each other.'));
         }
         else {
             var QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
             Modal.createDialog(QuestionDialog, {
-                title: "Warning",
+                title: _t('Warning'),
                 description:
                     <div>
-                        Resetting password will currently reset any end-to-end encryption keys on all devices,
-                        making encrypted chat history unreadable, unless you first export your room keys
-                        and re-import them afterwards.
-                        In future this <a href="https://github.com/vector-im/riot-web/issues/2671">will be improved</a>.
+                        { _t('Resetting password will currently reset any end-to-end encryption keys on all devices, making encrypted chat history unreadable, unless you first export your room keys and re-import them afterwards. In future this <a href="https://github.com/vector-im/riot-web/issues/2671">will be improved</a>') }.
```

Comment:
I'm not sure if wrapping is a good thing. This will make it more difficult for a script to auto-add strings to en_EN.json. But I've wrapped it anyway, and we can discuss this later...